### ISSUE TITLE
Help flag not working well

### ISSUE THREAD

**Issue by ktfth** [facebookexperimental/rome#264]
argv:1:7 flags/invalid ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Unknown help flag at 

    $ rome --help

**Comment by ktfth** [comment]
node version v13.12.0
npm version 6.14.4

**Comment by ktfth** [comment]
After try build again, this occur on the code base

 Building trunk 

 internal/fs/watchers.js:169:25 internalError/request ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ ENOSPC: System limit for number of file watchers reached, watch '/home/ktfth/war/rome/packages/@romejs/js-parser/test-fixtures/esprima/expression-primary-literal-regular-expression/u-flag-surrogate-pair'

  1.  FSWatcher.FSWatcher.<computed> (internal/fs/watchers.js:169:25)
  2.  Object.watch (fs.js:1388:33)
  3.  ___R$project$rome$$romejs$fs$index_ts$watch (scripts/vendor/rome.cjs:64308:58)
  4.  Object.onFoundDirectory (scripts/vendor/rome.cjs:89123:24)
  5.  ___R$project$rome$$romejs$core$master$fs$MemoryFileSystem_ts$default.addDirectory (scripts/vendor/rome.cjs:90062:13)
  6.  declareItem (scripts/vendor/rome.cjs:90075:23)
  7.  Function.all
  8.  ___R$project$rome$$romejs$core$master$fs$MemoryFileSystem_ts$default.addDirectory (scripts/vendor/rome.cjs:90088:8)
  9.  declareItem (scripts/vendor/rome.cjs:90075:12)
  10. Function.all
  11. ___R$project$rome$$romejs$core$master$fs$MemoryFileSystem_ts$default.addDirectory (scripts/vendor/rome.cjs:90088:8)
  12. declareItem (scripts/vendor/rome.cjs:90075:12)
  13. Function.all
  14. ___R$project$rome$$romejs$core$master$fs$MemoryFileSystem_ts$default.addDirectory (scripts/vendor/rome.cjs:90088:8)
  15. declareItem (scripts/vendor/rome.cjs:90075:12)
  16. Function.all
  17. ___R$project$rome$$romejs$core$master$fs$MemoryFileSystem_ts$default.addDirectory (scripts/vendor/rome.cjs:90088:8)
  18. declareItem (scripts/vendor/rome.cjs:90075:12)
  19. Function.all
  20. ___R$project$rome$$romejs$core$master$fs$MemoryFileSystem_ts$default.addDirectory (scripts/vendor/rome.cjs:90088:8)
      and 19 others...

  ⚠ This diagnostic was derived from an internal Rome error. The problem likely isn't with your code. Please report this if necessary ⚠

**Comment by linux-china** [comment]
Same problem to me. 

```
❯ rome --help

 argv:1:7 flags/invalid ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Unknown help flag at

    $ rome --help
           ^^^^^^

❯ node -v
v13.12.0
❯ npm -v
6.14.4
```

**Comment by sebmck** [comment]
Known issue #262 and #214

**Comment by sebmck** [comment]
Fixed by #262.
